examples/registration_form/lib/root/root_component.html

```diff
@@ -0,0 +1,57 @@
+<form #myForm="ngForm">
+    <section>
+        <material-input
+                type="text"
+                floatingLabel
+                label="First Name"
+                ngControl="first-name"
+                [(ngModel)]="firstName"
+                required>
+        </material-input>
+    </section>
+    <section>
+        <material-input
+                type="text"
+                floatingLabel
+                label="Last Name"
+                ngControl="last-name"
+                [(ngModel)]="lastName"
+                required>
+        </material-input>
+    </section>
+    <address-cmp></address-cmp>
+
+    <material-yes-no-buttons
+            saveCancel
+            yesHighlighted
+            raised
+            [yesDisabled]="!myForm.valid"
+            (yes)="onSubmit()"
```

Comment:
I'd pass in myForm here and the next line.

Reply:
Don't actually need it for onClear (yet), so left that out for now. I'll be adding it back when I support reset on forms.